docs/grammar.md

```diff
@@ -59,26 +59,37 @@ Terminals are used to match text into symbols. They can be defined as a combinat
 **Syntax:**
 
 ```html
-<NAME> [. <priority>] : <literals-and-or-terminals>
+<NAME> [. <priority>] : <items-to-match>
```

Comment:
"items" may be more correct, but to a newcomer it sounds ambiguous. What kind of items?

Perhaps below should be an enumeration of the kinds of items possible (i.e. rules, terminals, templates)